_layouts/country.html

```diff
@@ -92,12 +92,14 @@ <h4 id="contact-header"></h4>
         
         <!-- <h4 id="contact-header">{{ page.title }} has {{ projectcount }} HOT project{% if projectcount > 1%}s{%endif%}{% if memberscount > 0%} and {{ memberscount }} voting member{% if memberscount > 1%}s{%endif%}{%endif%}</h4> -->
         <div class="country-content top">
-          <p>Read more about each project below or learn more about <a href="/get-involved">getting involved</a>.</p>
+          <p>Read more about each project below or learn more about <a href="/get-involved" target="_blank">getting involved</a>.</p>
```

Comment:
This need not open in a new frame.

Reply:
Only external links need to open in a new tab/window. Other changes work good 👍

Reply:
@ramyaragupathy I have removed target="_blank" from getting involve link.